Documentation/commands.md

```diff
@@ -0,0 +1,23 @@
+# Commands
+
+This article outlines the commands exposed by the .NET Install Tool for Extension Authors. To see these commands used in an extension, check out the [sample extension](https://github.com/dotnet/vscode-dotnet-runtime/tree/master/sample) in this repository. Note that the majority of these commands are not user facing, meaning they can only be called programatically using the [VSCode API](https://code.visualstudio.com/api/extension-guides/command#programmatically-executing-a-command) by other extensions.
+
+## dotnet.acquire
+
+This command will install a .NET runtime. It accepts a [IDotnetAcquireContext](https://github.com/dotnet/vscode-dotnet-runtime/blob/master/vscode-dotnet-runtime-library/src/IDotnetAcquireContext.ts) object and returns the path to the .NET runtime executable. Note that the version value in IDotnetAcquireContext must be a valid major.minor runtime version, for example 3.1. The extension will automatically identify and install the latest patch of the provided version. It is generally recommended that extension authors call this command immedietly on every extension start up to ensure that the .NET runtime has been installed and is ready to use.
+
+## dotnet.uninstallAll
```

Comment:
Hmm, we probably shouldn't document this. Technically if an extension author has to do this then we've done something wrong. Moreover, given this extension is a singular extension that manages the .NET runtime we don't want to advertise ways other extensions can break experiences. Meaning, if extension A calls "uninstallAll" then extension B is indirectly affected.